.gitignore

```diff
@@ -7,4 +7,4 @@ resources/_gen
 public/
 
 .vscode/settings.json
-.DS_Store
\ No newline at end of file
+.DS_Store
```

Comment:
Hold on, why did the `resourcres/_gen` folder ever get added to begin with?

Reply:
idk, I did not manually create that.

Reply:
Nvm, i understood this change wrong while I was looking at it on my phone